os/compression/compress_read.c

```diff
@@ -39,7 +39,7 @@
  * Private Declarations
  ****************************************************************************/
 
-struct s_header compression_header;
+struct s_header *compression_header;
 struct s_buffer buffers;
```

Comment:
Please make them as `static` with different commit. They are used only in this file.

Reply:
Done. Created a separate commit for this.